### PR TITLE
Skip building ephemeral publish- branch

### PR DESCRIPTION
**Pull request by vogre**
On Jenkins in addition to CircleCI

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -15,6 +15,11 @@ node {
         ]),
     ]);
 
+    if ("${BRANCH_NAME}" =~ "^publish-.*") {
+        currentBuild.result = 'SUCCESS'
```
Comment (bergundy) [inline Jenkinsfile]: Is this required?

File: Jenkinsfile [hunk 2]
```diff
@@ -15,6 +15,11 @@ node {
...
+        return
```
Comment (bergundy) [inline Jenkinsfile]: Can haz semicolons?

File: Jenkinsfile (hunk 1 above)
Comment (vogre) [reply Jenkinsfile]: Seems to work without it.

File: Jenkinsfile (hunk 2 above)
Comment (vogre) [reply Jenkinsfile]: https://dzone.com/articles/groovy-sometimes-you-still

Groovy has fewer edge cases, but I will never remember them either.

**Pull request merged**